test/functional/km/control.cpp

```diff
@@ -479,6 +480,13 @@ Return Value:
         "[fnkt] FAIL: %S",
         Buffer);
 
+
+    PVOID Stack[16];
+    USHORT Frames = RtlCaptureStackBackTrace(0, 16, Stack, NULL);
+    for (USHORT i = 0; i < Frames; i++) {
+        DoTraceMessage(TRACE_FUNCTIONAL, "Stack frame %d: %p", i, Stack[i]);
```

Comment:
Ideally the stacks provide a module-relative address, else a list of raw kernel pointers will be very hard to use.

We will need to ask the kernel team to publicly document `ZwQuerySystemInformation(SystemModuleInformation)` which seems to be well-known already, and would allow us to query the module each address belongs to.

Reply:
Who is the kernel team contact?

Reply:
Or, instead, we could use `GlobalDriverObject->DriverStart` to determine our own driver start and range.